test/unit/operations/list_collections.test.js

```diff
@@ -29,7 +55,15 @@ describe('ListCollectionsOperation', function () {
       const operation = new ListCollectionsOperation(db, {}, { dbName: db });
 
       it('sets nameOnly to false', function () {
-        expect(operation.nameOnly).to.be.false;
+        expect(operation).to.have.property('nameOnly', false);
+      });
+    });
+
+    context('when authorizedCollections is not provided', function () {
```

Comment:
instead of this block, could we update the context in L54 to read `when no options are provided` and then move the `it` below into the expected block, since the setup is the same?